modules/client/shared/src/main/scala/lucuma/itc/client/ItcClient.scala

```diff
@@ -55,7 +55,7 @@ object ItcClient {
 
         val callOut: F[SpectroscopyResult] =
           for {
-            r  <- http.request(SpectroscopyQuery)(input)
+            r  <- http.request(SpectroscopyQuery).withInput(input)
```

Comment:
I tried to port this but I totally missed how to fix this line

Reply:
Actually, I just added this in clue so that this is easier now.